Pear/Controller/HomeViewController.swift

```diff
@@ -168,15 +171,22 @@ class HomeViewController: UIViewController {
 
     @objc private func toggleFeedbackMenu() {
         if displayMenu {
-            feedbackMenuView = FeedbackView(delegate: self)
+            guard let user = user, let superView = navigationController?.view else { return }
+            feedbackMenuView = OptionsView(
+                feedbackDelegate: self,
+                blockDelegate: self,
+                matchId: user.currentMatch?.id ?? 0,
+                blockId: user.currentMatch?.matchedUser.id ?? 0,
+                options: menuOptions,
+                superView: superView
+            )
             guard let feedbackMenuView = feedbackMenuView else { return }
-            feedbackMenuView.layer.cornerRadius = 20
             view.addSubview(feedbackMenuView)
 
             feedbackMenuView.snp.makeConstraints { make in
                 make.top.equalTo(feedbackButton.snp.bottom).offset(5)
                 make.trailing.equalTo(view.snp.trailing).offset(-25)
-                make.size.equalTo(CGSize(width: 150, height: 130))
+                make.size.equalTo(CGSize(width: 150, height: 43 * menuOptions.count))
```

Comment:
factor out constants into variables and do `.inset` instead of offsetting by a negative value.